.github/workflows/publish_client_library.yaml

```diff
@@ -0,0 +1,57 @@
+name: Publish package to Artifactory
+#on:
+#  release:
+#    types: [created]
+on:
+  push:
+    branches: [ yyu-CA-874 ]
```

Comment:
fix these before submitting. I'm also not sure we should do automatic library releases so much as manually trigger them. WDYT? Do workspace manager or jade-data-repo have an opinion?

Reply:
I know WM do that manually. I had this because CRL did this. Feels like we shouldn't have this automattly, sounds release and publishing package are not the same thing. Let me remove this.